src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/Methodologies/MethodologyRepositoryTests.cs

```diff
@@ -0,0 +1,310 @@
+                Publications = new List<Publication> {publication}
+            };
+
+            var contentDbContextId = Guid.NewGuid().ToString();
+            await using (var contentDbContext = DbUtils.InMemoryApplicationDbContext(contentDbContextId))
+            {
+                await contentDbContext.AddAsync(publication);
+                await contentDbContext.AddAsync(release);
+                await contentDbContext.AddAsync(userReleaseRole);
+                await contentDbContext.AddAsync(methodology);
+                await contentDbContext.SaveChangesAsync();
+            }
+
+            await using (var contentDbContext = DbUtils.InMemoryApplicationDbContext(contentDbContextId))
+            {
+                var methodologyRepository = new MethodologyRepository(contentDbContext);
+                Assert.True(
+                    await methodologyRepository.UserHasReleaseRoleAssociatedWithMethodology(userId, methodology));
+            }
+        }
+
+        [Fact]
+        public async Task GetMethodologiesForUser_ReleaseRoleLead()
+        {
+            var userId = Guid.NewGuid();
+            var publication = new Publication
+            {
+                Id = Guid.NewGuid()
+            };
+            var release = new Release
+            {
+                Id = Guid.NewGuid(),
+                PublicationId = publication.Id
+            };
+            var userReleaseRole = new UserReleaseRole
+            {
+                UserId = userId,
+                Release = release,
+                Role = ReleaseRole.Lead,
+            };
+            var methodology = new Methodology
+            {
+                Id = Guid.NewGuid(),
+                Title = "Test methodology",
+                Publications = new List<Publication> {publication}
+            };
+
+            var contentDbContextId = Guid.NewGuid().ToString();
+            await using (var contentDbContext = DbUtils.InMemoryApplicationDbContext(contentDbContextId))
+            {
+                await contentDbContext.AddAsync(publication);
+                await contentDbContext.AddAsync(release);
+                await contentDbContext.AddAsync(userReleaseRole);
+                await contentDbContext.AddAsync(methodology);
+                await contentDbContext.SaveChangesAsync();
+            }
+
+            await using (var contentDbContext = DbUtils.InMemoryApplicationDbContext(contentDbContextId))
+            {
+                var methodologyRepository = new MethodologyRepository(contentDbContext);
+                var methodologies = await methodologyRepository.GetMethodologiesForUser(userId);
+
+                Assert.Single(methodologies);
+                Assert.Equal(methodology.Id, methodologies[0].Id);
+                Assert.Equal(methodology.Title, methodologies[0].Title);
+            }
+        }
+
+        [Fact]
+        public async Task UserHasReleaseRoleAssociatedWithMethodology_ReleaseRolePrerelease()
```

Comment:
Would place this method next to the other `UserHasReleaseRoleAssociatedWithMethodology_` test cases so that they're all in the same place.